docs/device-management.md

```diff
@@ -1,18 +1,18 @@
 <h2 id="device-management">Device Management for Mbed OS</h2>
 
-This feature provides a way to add device management capabilities to Mbed OS devices using the Arm Pelion Device Management IoT platform.
+This feature provides a way to add device management capabilities to Mbed OS devices using the Pelion Device Management service.
 
 It:
 
 - Enables applications to connect and perform firmware updates in a few lines of code.
 - Runs separately from your main application; it does not take over your main loop.
-- Provides LWM2M resources, variables that sync automatically through Pelion Device Management.
-- Helps users avoid doing blocking network operations in interrupt contexts, by automatically defering actions to a separate thread.
-- Provides end-to-end Greentea tests for Pelion Device Management.
+- Provides LWM2M resources, variables that sync automatically through Device Management.
+- Helps users avoid doing blocking network operations in interrupt contexts, by automatically deferring actions to a separate thread.
+- Provides end-to-end Greentea tests for Device Management.
 
-This library makes it trivial to expose sensors, actuators and other variables to a cloud service. For a complete Pelion Device Management Client API, please see our [documentation](https://cloud.mbed.com/docs/current/mbed-cloud-client/index.html).
+This library makes it trivial to expose sensors, actuators and other variables to a cloud service. For a complete Device Management Client API, please see our [documentation](https://cloud.mbed.com/docs/current/mbed-cloud-client/index.html). <!---This isn't a link to API references, though.--->
```

Comment:
Could you suggest a link in its place?

Reply:
https://cloud.mbed.com/docs/current/client-api-references/index.html